cfgov/unprocessed/apps/youth-employment-success/js/yes-store.js

```diff
@@ -0,0 +1,17 @@
+import Store from '../../../js/organisms/MortgagePerformanceTrends/stores/store';
+import { UNDEFINED } from './util';
+
+class YesStore extends Store {
+  constructor( reducer, middlewares ) {
+    super( middlewares );
+
+    this.reducer = reducer;
+    this.state = this.reduce( UNDEFINED );
```

Comment:
If you're using undefined only once, probably just have `let UNDEFINED;` above this line and get rid of the import at the top.

Reply:
Here and elsewhere. We used to have `UNDEFINED` in a utils but ended up just defining it where it was needed as it was only used once usually.